dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesRun.java

```diff
@@ -0,0 +1,390 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.dsl.jbang.core.commands.kubernetes;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import org.apache.camel.RuntimeCamelException;
+import org.apache.camel.dsl.jbang.core.commands.CamelJBangMain;
+import org.apache.camel.dsl.jbang.core.commands.kubernetes.traits.BaseTrait;
+import org.apache.camel.dsl.jbang.core.common.RuntimeCompletionCandidates;
+import org.apache.camel.dsl.jbang.core.common.RuntimeType;
+import org.apache.camel.dsl.jbang.core.common.RuntimeTypeConverter;
+import org.apache.camel.impl.DefaultCamelContext;
+import org.apache.camel.support.FileWatcherResourceReloadStrategy;
+import org.apache.camel.util.FileUtil;
+import org.apache.camel.util.IOHelper;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "run", description = "Run Camel application on Kubernetes", sortOptions = false)
+public class KubernetesRun extends KubernetesBaseCommand {
+
+    @CommandLine.Parameters(description = "The Camel file(s) to run.",
+                            arity = "0..9", paramLabel = "<files>")
+    String[] filePaths;
+
+    @CommandLine.Option(names = { "--trait-profile" }, description = "The trait profile to use for the deployment.")
+    String traitProfile;
+
+    @CommandLine.Option(names = { "--dependency", "-d" },
+                        description = "Adds dependency that should be included, use \"camel:\" prefix for a Camel component, \"mvn:org.my:app:1.0\" for a Maven dependency.")
+    String[] dependencies;
+
+    @CommandLine.Option(names = { "--property", "-p" },
```

Comment:
We do not favour single char options, we use `--prop` and `--dep` / `--deps`
See the vanilla run command